frontend/apps/w3bstreaming/package.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "@craco/craco": "^7.1.0",
+    "@types/lodash.merge": "^4.6.7",
+    "@types/node": "18.17.15",
+    "@types/react": "18.2.24",
+    "@types/react-datepicker": "^4.11.2",
+    "@types/react-dom": "18.2.8",
+    "@types/react-gtm-module": "2.0.1",
```

Comment:
please use error tracking and analytics utils from shared packages, remove direct dependencies tied to it